### PR TITLE
replaced all inputs with our component

### PR DESCRIPTION
**Pull request by joelimgu**
Don't look at the first two commits, it's just me doing stupid things

### CODE REVIEW

File: src/components/inputs/input.tsx [hunk 1]
```diff
@@ -1,5 +1,42 @@
 import styled from "styled-components";
 
 // Use these in the event we want to modify them later
-export const TextArea = styled.textarea``;
-export const Input = styled.input``;
+export const TextArea = styled.textarea`
+background: transparent;
```
Comment (lucemans) [inline src/components/inputs/input.tsx:5]: Indentation

File: src/components/inputs/input.tsx [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+padding: 1rem 1rem;
+box-shadow: rgb(0 0 0 / 10%) -3px 4px 14px;
+border-color: ${({ theme }) => theme.palette.primary[700]};
+background: ${({ theme }) => theme.palette.primary[700]};
+border-radius: ${({ theme }) => theme.borderRadius};
+flex: 1;
+
+&:placeholder {
+    color: ${({ theme }) => theme.palette.primary[300]};
+}
```
Comment (lucemans) [inline src/components/inputs/input.tsx:41]: Indentation

File: src/components/rooms/Rooms.tsx [hunk 3]
```diff
@@ -13,8 +13,11 @@ import { notDraggable } from "../../library/mixin/mixin";
 import Modal from "../../library/portals/Modal";
 import { Button } from "../button/Button";
 import { Card as CardOriginal } from "../card/Card";
+import { Input, TextArea } from "../inputs/input";
 import { NoRooms } from "../logo/NoRooms";
 
+
+
```
Comment (lucemans) [inline src/components/rooms/Rooms.tsx:20]: Ahhhhhhhhhhhh

**Review by lucemans — CHANGES_REQUESTED** [review]
Please change this and then LGTM

**Review by joelimgu — COMMENTED** [review]
yep that's better

File: src/components/search/Search.tsx [hunk 4]
```diff
@@ -62,7 +64,7 @@ export const Search: FC = () => {
     return (
         <Wrapper>
             <Box>
-                <Input placeholder="Search for rooms, users or categories" onChange={(event) => {setSearchResults(event.target.value);}}/>
+                <SearchInput placeholder="Search for rooms, users or categories" onChange={(event) => {setSearchResults(event.target.value);}}/>
```
Comment (chalkedgoose) [inline src/components/search/Search.tsx:67]: Nit: A anonymous function without curly braces would work fine here.

**Review by chalkedgoose — APPROVED** [review]

**Review by 4nkitd — APPROVED** [review]
LGTM!

**Review by joelimgu — COMMENTED** [review]
ok

**Comment by lucemans** [comment]
The changes that were requested were never made, @joelimgu please have another look at this.

**Review by lucemans — APPROVED** [review]
LGTM!

**Pull request merged**